tests/nonlinear_channel_flow_iterated_IMPES/solution/solution-00000.0000.gnuplot

```diff
@@ -0,0 +1,1799 @@
+# This file was generated by the deal.II library.
```

Comment:
I would argue that we do not need the visualization output for both new tests. Please add the statistics output for both tests, and remove the visualization output for iterated IMPES (keep it for the Newton test).